package.json

```diff
@@ -1,31 +1,32 @@
 {
   "devDependencies": {
-    "apache-server-configs": "^2.7.1",
-    "babel-core": "^5.4.7",
-    "browser-sync": "^2.9.0",
-    "del": "^1.1.0",
+    "apache-server-configs": "^2.14.0",
+    "babel-core": "^6.0.12",
+    "babel-preset-es2015": "^6.0.14",
+    "browser-sync": "^2.9.11",
+    "del": "^2.0.2",
     "gulp": "^3.9.0",
-    "gulp-autoprefixer": "^2.0.0",
-    "gulp-babel": "^5.2.0",
-    "gulp-cache": "0.2.2",
-    "gulp-concat": "^2.5.2",
-    "gulp-if": "^1.2.1",
-    "gulp-imagemin": "^2.0.0",
-    "gulp-jshint": "^1.6.3",
-    "gulp-load-plugins": "^0.10.0",
-    "gulp-minify-css": "^1.1.6",
-    "gulp-minify-html": "0.1.5",
-    "gulp-newer": "^0.5.1",
-    "gulp-sass": "^2.0.0",
-    "gulp-size": "^1.0.0",
-    "gulp-sourcemaps": "^1.3.0",
-    "gulp-uglify": "^1.0.1",
-    "gulp-uncss": "^1.0.0",
-    "gulp-useref": "^1.0.1",
-    "jshint-stylish": "^1.0.0",
-    "psi": "^1.0.4",
+    "gulp-autoprefixer": "^3.1.0",
+    "gulp-babel": "^6.0.0",
+    "gulp-cache": "^0.3.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-if": "^2.0.0",
+    "gulp-imagemin": "^2.4.0",
+    "gulp-jshint": "^1.12.0",
+    "gulp-load-plugins": "^1.0.0",
+    "gulp-minify-css": "^1.2.1",
+    "gulp-minify-html": "^1.0.4",
+    "gulp-newer": "^1.0.0",
+    "gulp-sass": "^2.1.0",
+    "gulp-size": "^2.0.0",
+    "gulp-sourcemaps": "^1.6.0",
+    "gulp-uglify": "^1.4.2",
+    "gulp-uncss": "^1.0.4",
+    "gulp-useref": "^2.0.0",
+    "jshint-stylish": "^2.0.1",
+    "psi": "^1.0.6",
     "require-dir": "^0.3.0",
-    "run-sequence": "^1.0.1",
+    "run-sequence": "^1.1.4",
     "sw-precache": "^2.2.0"
```

Comment:
Thanks, I didn't know that. I will restore back all unnecessary version increments.